central/sensor/service/pipeline/enhancements/pipeline.go

```diff
@@ -36,7 +36,6 @@ func GetPipeline() pipeline.Fragment {
 
 func (p pipelineImpl) OnFinish(_ string) {}
 
-// TODO(ROX-21202): Add capabilities
 func (p pipelineImpl) Capabilities() []centralsensor.CentralCapability {
```

Comment:
Unrelated to your changes, but why we don't have a pointer receiver `func (p *pipelineImpl)` here?

Reply:
I think that might have been the code generator, I've rectified this :)

Reply:
Yeah, I think it would be good to correct that. Keeping it without the pointer would mean that the methods cannot modify the fields of `p` ([toy example](https://go.dev/tour/methods/4))

Reply:
See [523430b](https://github.com/stackrox/stackrox/pull/9306/commits/523430b2351666b45a166cb69c181c9cccf644f9)